public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -127,21 +127,23 @@ <h5 class="col-xs-12 box-title">
           </div>
         </div>
         <div class="row">
-          <div class="col-md-4 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md-4 grid-header flex-xs-middle"> Number of particpants </div>
-          <div class="col-md-3 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
+          <div class="col-md-3 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
+          <div class="col-md-2 grid-header flex-xs-middle"> Concept Id </div>
+          <div class="col-md-4 grid-header flex-xs-middle"> Number of participants </div>
+          <div class="col-md-2 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
```

Comment:
I think this column the source vocab and code shold be md-4 and the num participants should be md-2